createlinks-virtualhosts

```diff
@@ -138,3 +138,15 @@ for my $event (qw(
     event_services($event, 'rh-php73-php-fpm' => 'restart');
     event_services($event, 'php80-php-fpm' => 'restart');
 }
+
+#--------------------------------------------------
+# templates for trusted-networks-modify event
+#--------------------------------------------------
+
+event_templates('trusted-networks-modify', qw(
+             /etc/httpd/conf.d/virtualhosts.conf
+));
+
+event_services('trusted-networks-modify', qw(
+               httpd reload
+));
```

Comment:
The `httpd` restart is already present inside the default package: https://github.com/NethServer/nethserver-httpd/blob/master/createlinks-default#L28

```suggestion
```